packages/paste-core/layout/grid/src/utils.tsx

```diff
@@ -13,33 +12,31 @@ import {
 type Vertical = ResponsiveValue<boolean>;
 
 // Gets the gutter and returns the value to be used as negative margin to Grid
-export const getOuterGutterPull = (theme: ThemeShape, gutter?: Space): Margin => {
+export const getOuterGutterPull = (gutter?: Space): Margin => {
   if (Array.isArray(gutter)) {
     return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
-      return `-${theme.space[value]}` as SpaceOptions;
+      if (value !== 'space0') {
+        return value.replace('space', 'spaceNegative') as SpaceOptions;
+      }
+      if (value === 'space0') {
+        return 'space0' as SpaceOptions;
+      }
+      return null;
     });
   }
 
-  if (gutter) {
-    return `-${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
-  }
-
-  return 'auto';
-};
-
-// Gets the gutter and returns the value to be used as padding for Columns
-export const getColumnGutters = (theme: ThemeShape, gutter?: Space): Padding => {
-  if (Array.isArray(gutter)) {
-    return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
-      return `${theme.space[value]}` as SpaceOptions;
-    });
+  if (gutter === 'space0') {
+    return 'space0';
   }
 
   if (gutter) {
-    return `${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
+    // Ignoring this ts error: Type '"space0"' has no call signatures.
+    // We're checking for space0 already above.
+    // @ts-ignore
+    return gutter.replace('space', 'spaceNegative') as SpaceOptions;
```

Comment:
For whatever reason typescript thinks gutter _might_ be a number

<img width="1847" alt="Screen Shot 2020-07-22 at 10 39 17" src="https://user-images.githubusercontent.com/368249/88210246-bb283300-cc08-11ea-8b40-2d4af91f4467.png">

if you add `&& typeof gutter === 'string'` to the condition, the error seems to go away

Reply:
That is a really odd one, since we're setting the type to `Space`.